### PR TITLE
fix: overlays render properly and can close with onEscape

### LINKED ISSUE #672: Rename warning overlay is broken

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #672 

This fixes the bug where overlays were rendering off to the side instead of as overlays
It also adds onEscape functionality, which is nice
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

**Comment by mavarius** [comment]
Closes #668

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 42 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

File: src/organizations/components/RenameOrgOverlay.tsx [hunk 1]
```diff
@@ -15,17 +15,31 @@ import {ErrorHandling} from 'src/shared/decorators/errors'
 class RenameOrgOverlay extends PureComponent<
   RouteComponentProps<{orgID: string}>
 > {
+  state = {
+    visible: false,
+  }
+  componentDidMount() {
+    this.setState({visible: true})
```
Comment (drdelambre) [inline src/organizations/components/RenameOrgOverlay.tsx:22]: why is this state stuff here? doesn't the handleClose unmount the component? if so, please remove, if not, please ignore

File: src/buckets/components/RenameBucketOverlay.tsx [hunk 2]
```diff
@@ -10,22 +10,37 @@ import RenameBucketForm from 'src/buckets/components/RenameBucketForm'
...
+  state = {
+    visible: false,
+  }
+
+  componentDidMount() {
+    this.setState({visible: true})
+  }
+
+  private closeOverlay = () => {
+    this.setState({visible: false})
```
Comment (drdelambre) [inline src/buckets/components/RenameBucketOverlay.tsx:27]: same state stuff here

File: src/organizations/components/RenameOrgOverlay.tsx (hunk 1 above)
Comment (ShmuelLotman) [reply src/organizations/components/RenameOrgOverlay.tsx:22]: its to manage the visible state and expose the onEscape

**Review by drdelambre — APPROVED** [review]
thanks for cleaning up the state

**Comment by ShmuelLotman** [comment]
Merging. The last check passed (unit) but github didnt update

**Pull request merged**